docs/scripts/slides.js

```diff
@@ -50,8 +50,8 @@ function testingSlides() {
     '30-testing/00-TITLE.md',
     '30-testing/00-testing.md',
     '30-testing/01-TITLE.md',
-    '30-testing/01-javascript.md',
     '30-testing/01-others.md',
+    '30-testing/01-javascript.md',
```

Comment:
The first slide of `01-others.md` is designed as the following of the `01-javascript.md`.

If we change the order of slides, we will need some rewrites. 

What is the goal behind this slide switch?

Reply:
I was wonderring the fact that it could be better to show the global testing like bats before the specific like JS 
But now i've re-read the slides, it look like it doesn't make sens with the last slide 

i've made a revert of this change